src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -341,7 +342,12 @@ const ActionsPane = (props: {contentHeight: string}) => {
   }, [selectedResourceId, resourceMap]);
 
   useEffect(() => {
```

Comment:
missing `selectedPath` as dependency in this `useEffect`

Reply:
adding this breaks the source editor (it doesn't show up - not sure why)

Reply:
sorry - my bad - you're right!